docs/visual-basic/programming-guide/language-features/procedures/how-to-call-an-event-handler.md

```diff
@@ -1,25 +1,26 @@
 ---
-title: "How to: Call an Event Handler"
+title: "How to call an event handler"
 ms.date: 07/20/2015
 helpviewer_keywords:
   - "Visual Basic code, procedures"
   - "event handlers [Visual Basic], calling"
   - "event handlers"
   - "procedures [Visual Basic], event handlers"
   - "procedures [Visual Basic], calling"
+no-loc: [Handles, WithEvents]
```

Comment:
@JasonCard is no-loc case sensitive?

There is one occurrence of lowercase "handles" that **should be translated**.

If it's not case sensitive, then I'll have to manually go through the "Handles" that should **not be translated**, and apply inline :::no-loc::: to them.

Reply:
@Youssef1313 Yes, no-loc metadata is case sensitive, so the one occurrence of lowercase "handles" will be translated.